packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
+## <a id="dabs"></a>Deploying Databricks Asset Bundle
+
+### <a id="what-is-dab"></a>What is a Databricks Asset Bundle?
+
+Databricks Asset Bundles make it possible to describe Databricks resources such as jobs, pipelines, and notebooks as source files. These source files provide an end-to-end definition of a project, including how it should be structured, tested, and deployed, which makes it easier to collaborate on projects during active development. For more information, see [Databricks Asset Bundle](https://docs.databricks.com/en/dev-tools/bundles/index.html).
+
+### <a id="deploy-dab"></a>Deploying a Databricks Asset Bundle?
+
+1. In the Databricks panel, look for the "Bundle Resource Explorer" view.
+2. Click on the "Deploy" button.
+3. You can monitor the deployment status in the log output window.
+
+### <a id="deploy-run-job-pipeline"></a>Run a Job or Pipeline
+
+You can run a job or a pipeline managed by Databricks Asset Bundles, from the "Bundle Resource Explorer" view.
+
+1. In the Databricks panel, look for the "Bundle Resource Explorer" view.
+2. Hover on the line with the job or pipeline you want to run.
+3. Click on the "Run" button.
+
+This will deploy the Databricks Asset Bundle and run the job or pipeline. You can monitor the run progress in the output terminal window. You can also open the run, job or pipeline in workspace by clicking on the "Open link externally" button.
+
+#### Use interactive cluster for running jobs
+
+By default, a job is run using a jobs cluster. You can change this behaviour and use the interactive cluster selected previously ([Select a cluster](#select-cluster)) to run the job.
+
+1. In the Databricks panel, look for the "Configuration" view.
+2. Check the "Override Jobs cluster in bundle" checkbox.
 
-Both ways of running code on a cluster are also available in custom run configurations. In the "Run and Debug" panel you can click "Add configuration..." and select either "Databricks: Launch" or "Databricks: Launch as Workflow". Using run configuration you can also pass in command line arguments and run your code by simply pressing `F5`.
+# <a id="changes-from-v1"></a> Key behaviour changes for users of Databricks Extension v1
 
-![configure](./images/custom-runner.gif)
+## <a id="migrate-from-v1"></a>Migrate a project from Databricks Extension v1 to v2
 
-## <a id="settings"></a>Extension Settings
+If you are using Databricks Extension v1, will automatically migrate your project to v2 when you open it in v2. The migration process will create a new [`databricks.yml`](#what-is-databricksyaml) file in the root of your project and move the configurations from the old `.databricks/project.json` to the new `databricks.yml` file.
 
-This extension contributes the following settings:
+> **Note**: This means that you will start seeing a `databricks.yml` file in your project root directory and in your VCS change logs. We recomment comitting this file to your VCS.
```

Comment:
What is VCS? I assume version control system, but can we spell it out?

(Also typos "We recommend committing this file to your VCS.")